R/get_param.R

```diff
@@ -28,11 +28,9 @@ pluck_param <- function(x, file, if_null) {
   }
 
   if (is.null(out) && !is.null(if_null)) {
-    param_ <- ui_field(param)
     warn(glue("On config.yml, {param} is undefined."))
 
-    if_null_ <- ui_field(if_null)
-    inform(glue("Setting {param_} to default value: {if_null_}."))
+    inform(glue("Setting {param} to default value: {if_null}."))
```

Comment:
Sorry for mixing the removal of `ui_field()` with the refactoring of variables (here removing the intermediate variable `if_null`).

Reply:
The `ui_*` helpers are tagged as internal, but the helpfile should pop up with `?usethis::ui`. You like colour messages, so you may like these helpers

![image](https://user-images.githubusercontent.com/5856545/102001506-e1429280-3cd1-11eb-98a9-8f6de5c37474.png)